spec/models/collection_draft_spec.rb

```diff
@@ -232,4 +232,66 @@
     collection_draft = create(:collection_draft)
     expect(collection_draft.update(collection_concept_id: 'C12345-MMT_2')).to eq(false)
   end
+
+  # Keyword Recommendations
+  context 'when Keyword Recommendations is turned off' do
+    before do
+      allow(Mmt::Application.config).to receive(:gkr_enabled).and_return(false)
+    end
+
+    it '"keyword_recommendation_needed?" returns `nil`' do
+      collection_draft = create(:full_collection_draft)
+      expect(collection_draft.keyword_recommendation_needed?).to eq(nil)
+    end
+
+    it '"record_recommendation_provided" returns `nil`' do
+      collection_draft = create(:full_collection_draft)
+      expect(collection_draft.record_recommendation_provided).to eq(nil)
+    end
+  end
+
+  context 'when Keyword Recommendations is turned on' do
+    before do
+      allow(Mmt::Application.config).to receive(:gkr_enabled).and_return(true)
```

Comment:
If test is true in the config, why do we need these?

Reply:
I don't know that it will, but if the previous context block gets run first (mocking it to be `false`), it could interfere with what is in the actual config, so this is just to make sure we have the setting we want.